index.html

```diff
@@ -197,6 +197,7 @@ <h2 class="contact-header">Contact me</h2>
                 class="input name-input"
                 maxlength="30"
                 required
+                onchange="changeEvent()"
```

Comment:
- [ ] [OPTIONAL] I highly advise you to handle events only in your JavaScript as mixing JS and HTML is not considered good practice. Think of it as a separation of concern. The markup is there to provide the client with a UI. The JS is there to handle anything related to the behavior of your webpage. ✔